modules/gatsby/src/components/layout/header.jsx

```diff
@@ -121,7 +121,7 @@ export default class Header extends Component {
               }
             ]
           : []),
-        ...(props.config && props.config.ADDITIONAL_LINKS)
+        ...(props.config && props.config.ADDITIONAL_LINKS.map(link => ({...link, label: link.name})))
```

Comment:
It gets a bit hard to read with all these nested expressions. Maybe create each sublist in advance and assign to a var and just do ```, ...GITHUB_LINKS, ...ADDITIONAL_LINKS]```.

Also since this is getting long, recommend breaking out a function `generateHeaderLinks()`.